src/index.js

```diff
@@ -7,14 +7,14 @@ consoleStamp(console);
 import scrape from './scraper/index.js';
 import { persistRaw, persistSanitized, pushChanges } from './history/index.js';
 import sanitize from './sanitizer/index.js';
-import getServiceProviders from './service_providers/index.js';
+import getServiceProviders, { getSanitizers } from './service_providers/index.js';
 import { DOCUMENTS_TYPES } from './documents_types.js';
 
-export async function updateServiceProviderDocument(serviceProviderId, serviceProviderName, documentType, documentUrl, documentContentSelector) {
+export async function updateServiceProviderDocument({ serviceProviderId, serviceProviderName, documentType, documentParams }) {
```

Comment:
To me this is not the document _parameters_ but really the _document_ itself —or maybe the _document descriptors_?

```suggestion
export async function updateServiceProviderDocument({ serviceProviderId, serviceProviderName, documentType, document }) {
```